### PR TITLE
Updating istio dependencies

### PR DESCRIPTION
**Pull request by didierofrivia**
as part of https://github.com/Kuadrant/kuadrant-operator/pull/303

### CODE REVIEW

**Comment by eguzki** [comment]
looking good

File: controllers/ratelimitpolicy_controller_test.go [hunk 1]
```diff
@@ -416,7 +416,7 @@ var _ = Describe("RateLimitPolicy controller", func() {
 	})
 
 	Context("RLP targeting Gateway", func() {
-		It("Creates all the resources for a basic Gateway and RateLimitPolicy", func() {
+		FIt("Creates all the resources for a basic Gateway and RateLimitPolicy", func() {
```
Comment (alexsnaps) [inline controllers/ratelimitpolicy_controller_test.go:419]: TIL! 🙏

Comment (eguzki) [reply controllers/ratelimitpolicy_controller_test.go:419]: do not forget to remove that :exclamation:

**Comment by alexsnaps** [comment]
😩 I don't, not failing on my computer… for… reasons! So I restart it here and it isn't failing either 😢

**Review by eguzki — APPROVED** [review]

**Pull request merged**